### PR TITLE
added documentation for easy access and changed documentation error

### PR DESCRIPTION
**Pull request by skjerns**
deals with the suggestions at #4398 and #4381

### CODE REVIEW

File: doc/manual/memory.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ MNE-Python can read data on-demand using the ``preload`` option provided in :ref
     raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
     raw = io.read_raw_fif(raw_fname, preload=False)
 
-.. note:: Filtering does not work with ``preload=False``.
+.. note:: Filtering, resampling and dropping or selection channels does not work with ``preload=False``.
```
Comment (Eric89GXL) [inline doc/manual/memory.rst]: this is a case where the Oxford comma really helps (also there is a typo)
"Filtering, resampling, and dropping or selecting channels ..."

File: doc/manual/memory.rst [hunk 2]
```diff
@@ -44,3 +44,11 @@ To explicitly reject artifacts with ``preload=False``, use the function :func:`m
 Loading data explicitly
 =======================
 To load the data if ``preload=False`` was initially selected, use the functions :func:`mne.io.Raw.load_data` and :func:`mne.Epochs.load_data`.
+
+Simplest way to access data
+===========================
+If you just want your raw data as a numpy array to work with it in a different framework you can use slicing syntax::
+
+    first_channel, first_channel_times = raw[0,:]
```
Comment (agramfort) [inline doc/manual/memory.rst]: first_channel_data, times = raw[0,:] 

times is the same for all channels

Comment (mmagnuski) [inline doc/manual/memory.rst]: also - pep8 (`raw[0, :]`) (space before colon)

File: examples/preprocessing/plot_resample.py [hunk 3]
```diff
@@ -61,7 +61,7 @@
 # When resampling epochs is unwanted or impossible, for example when the data
 # doesn't fit into memory or your analysis pipeline doesn't involve epochs at
 # all, the alternative approach is to resample the continuous data. This
-# can also be done on non-preloaded data.
+# can only be done on loaded or pre-loaded data.
```
Comment (mmagnuski) [inline examples/preprocessing/plot_resample.py]: "loaded or pre-loaded" - what is the difference?

Comment (Eric89GXL) [inline examples/preprocessing/plot_resample.py]: None in practice, as we have `preload=True` and `inst.load_data()` that use different naming schemes to effectively accomplish the same things. Can you think of a clearer wording? Maybe "on data that has been loaded from disk into memory or a memmapped file, e.g. using `preload` keyword argument or `inst.load_data()`.", but that's a bit more verbose.

**Comment by agramfort** [comment]
finished in feae9f63bd20b7911e033c126426cfef1d04d0b6

**Pull request closed without merging**